src/Illuminate/Database/Query/Builder.php

```diff
@@ -1299,6 +1300,169 @@ public function orWhereNotNull($column)
         return $this->whereNotNull($column, 'or');
     }
 
+    /**
+     * Add a where clause to determine if "date" column is in the past to the query.
+     *
+     * @param  string|array  $columns
+     * @param  string  $boolean
+     * @param  bool  $not
+     * @return $this
+     */
+    public function wherePast($columns, $boolean = 'and', $not = false)
+    {
+        $type = 'Basic';
+        $operator = $not ? '>=' : '<';
+        $value = Carbon::now()->format('Y-m-d h:i:s');
```

Comment:
`Y-m-d h:i:s` should be `Y-m-d H:i:s` with a capital H, I believe.

Small h is 12 hour clock: https://www.php.net/manual/en/datetime.format.php

Reply:
Thanks. Just pushed a patch which should fix it. As well as add some test variance.